### DISCUSSION TITLE
Table In Postgres

### DISCUSSION THREAD

**Discussion by sjarifHD** [Budibase/budibase#4186] [Help] [answered]
Hi, I installed budibase self hosted and connected to existing postgres database. I succesfully add new table and inserting data via budibase. But I can't found my new table when accessing my postgres database directly. What I need to check? Thank you....

**Comment by hendrikjap** [comment, marked as answer]
Kindly check whether you are accessing the correct schema. Usually it is in public.tblxxx

**Reply by sjarifHD** [reply]
I think I use correct schema, because I can fetch existing table from my databases.
